### PR TITLE
Add layer5 in the news section 

### LINKED ISSUE #1604: [Proudly Maintaining] Text escaping on Image Hub card
**Description**
On the home page, in the [proud maintainers section](https://github.com/layer5io/layer5/tree/master/src/sections/Home/Proud-maintainers), the label "Image Hub" is escaping its card.

**Expected Behavior**
Labels should be contained to the card. They will need to shrink or disappear as the screen width is reduced.

**Screenshots**
Screen Shot 2021-02-24 at 7 01 22 AM

**Environment:**
- Browser: Safari

---

[Optional] **To Reproduce**

[Optional] **Additional Context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**

This PR also fixes #1604 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js [hunk 1]
```diff
@@ -83,18 +83,22 @@ const ProjectItemWrapper = styled.section`
 			padding: 1.5rem 1rem;
 			max-height: 125px;
 		}
-	}
+      @media screen and (max-width: 550px) {
+          img {
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Proud-maintainers/proudMaintainers.style.js:87]: Don't we wanr them to be specific to the `.proj_icon_2` class? Like the prevycode?

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js [hunk 2]
```diff
@@ -83,18 +83,22 @@ const ProjectItemWrapper = styled.section`
...
-		.proj_icon_2 {
-			img {
-				padding: 1.65rem 0.4rem;
-			}
-			h4 {
-				font-size: 1rem;
-			}
-		}
+      @media screen and (max-width: 460px){
+        h4 {
```
Comment (Nikhil-Ladha) [inline src/sections/Home/Proud-maintainers/proudMaintainers.style.js:96]: Same as above

File: src/sections/Home/Proud-maintainers/proudMaintainers.style.js (hunk 1 above)
Comment (Tanuj22) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:87]: It is inside the `.proj_icon_2` class

Comment (Nikhil-Ladha) [reply src/sections/Home/Proud-maintainers/proudMaintainers.style.js:87]: Ohhh, I missed that curly brace 😂

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**